services/compliance/continuous_compliance_policy/continuous_compliance_policy.go

```diff
@@ -6,24 +6,24 @@ import (
 )
 
 const (
-	continuousComplianceResourcePath = "Compliance/ContinuousCompliancePolicy"
+	continuousComplianceResourcePath = "ContinuousCompliancePolicyV2"
 )
 
 type ContinuousCompliancePolicyRequest struct {
-	CloudAccountID    string   `json:"cloudAccountId"`
-	ExternalAccountID string   `json:"externalAccountId"`
-	CloudAccountType  string   `json:"cloudAccountType,omitempty"`
-	BundleID          int      `json:"bundleId"`
+	TargetId 		  string   `json:"targetId"`
+	TargetType 		  string   `json:"targetType,omitempty"`
+	RulesetId         int      `json:"rulesetId"`
```

Comment:
you should run formatter on every file or tests will fail. 
you can do it either `go fmt` in console or use the formatter in GoLand

Reply:
Weird, in GoLand IDE it's look aligned.